gravitee-apim-console-webui/src/management/api/api-traffic-v4/analytics/api-analytics-native/api-analytics-native.component.html

```diff
@@ -18,7 +18,7 @@
 <api-analytics-native-filter-bar
   [activeFilters]="activeFilters()"
   [plans]="apiPlans$ | async"
-  [applications]="applications$ | async"
+  [applications]="applications"
```

Comment:
what about keep `| async` ? 
In Angular, it is good practice to try to keep observables until an async pipe :)